portal-web/test/functional/com/liferay/portalweb/macros/CrossClusterReplication.macro

```diff
@@ -47,9 +47,16 @@ definition {
 
 		FormFields.enableCheckbox(fieldName = "ccrEnabled");
 
-		FormFields.editTextMultiline(
-			fieldName = "ccrLocalClusterConnectionConfigurations",
-			fieldValue = "localhost:8080,localcluster");
+		if (${portalCluster} == "true") {
+			FormFields.editTextMultiline(
+				fieldName = "ccrLocalClusterConnectionConfigurations",
+				fieldValue = "localhost:9080,localcluster");
```

Comment:
Define both `localhost:9080,localcluster` and `localhost:8080,localcluster` when portal is clustered